.bazelrc

```diff
@@ -136,6 +136,7 @@ build --host_java_toolchain=//third_party/toolchains/java:tf_java_toolchain
 # environment variable "TF_MKL_ROOT" every time before build.
 build:mkl --define=build_with_mkl=true --define=enable_mkl=true
 build:mkl --define=tensorflow_mkldnn_contraction_kernel=0
+build:mkl --define=build_with_mkl_dnn_v1_only=true
```

Comment:
Now that we have time, can we separate this to another PR? Just so it's cleaner/easier for debugging.

Reply:
@penpornk should I submit separate PR now? or wait for this PR to approve and merge?